Main.cpp

```diff
@@ -421,26 +423,24 @@ bool PlayIntroTitle() {
     BITMAP **apStarLargeBitmaps = starLargeFile.GetAsAnimation(starLargeBitmapCount);
     BITMAP **apStarHugeBitmaps = starHugeFile.GetAsAnimation(starHugeBitmapCount);
     Star *aStars = new Star[starCount];
-    StarSize size;
 
     for (int star = 0; star < starCount; ++star) {
         if (RandomNum() < 0.95F) {
-            aStars[star].m_Size = StarSmall;
+			// Default size is StarSmall.
             aStars[star].m_pBitmap = apStarSmallBitmaps[RandomNum(0, starSmallBitmapCount - 1)];
-            aStars[star].m_Intensity = RandomNum(0.001F, 0.5F);
+			aStars[star].m_Intensity = RandomNum(0, 92);
         }
         else if (RandomNum() < 0.85F) {
             aStars[star].m_Size = StarLarge;
             aStars[star].m_pBitmap = apStarLargeBitmaps[RandomNum(0, starLargeBitmapCount - 1)];
-            aStars[star].m_Intensity = RandomNum(0.6F, 1.0F);
+			aStars[star].m_Intensity = RandomNum(111, 185);
         }
         else {
             aStars[star].m_Size = StarHuge;
             aStars[star].m_pBitmap = apStarHugeBitmaps[RandomNum(0, starLargeBitmapCount - 1)];
-            aStars[star].m_Intensity = RandomNum(0.9F, 1.0F);
+            aStars[star].m_Intensity = RandomNum(166, 185);
         }
-        aStars[star].m_Pos.SetXY(resX * RandomNum(), pBackdrop->GetBitmap()->h * RandomNum());//resY * RandomNum());
-        aStars[star].m_Pos.Floor();
+		aStars[star].m_Pos = std::array<int, 2>{RandomNum(0, resX), RandomNum(0, pBackdrop->GetBitmap()->h)};
```

Comment:
Wonder why this uses resX but not resY? Is the backdrop height different from the res height?

Reply:
The res vars describe the game window, so if it has a titlebar then it's not what we want for drawing area.
Then why not use the bitmap width instead of resX? Because the bitmap is only 640x1280.
Then what happens in resolutions with height more than 1280? They have a empty horizontal strip.
Actually this happens to me during the intro even on 1920x1080 game-resolution.
The starry background scrolls upwards and reveals an empty horizontal strip at the bottom.
So actually even using the bitmap's height is inadequate if the game runs at a resolution with Y higher than about 1080.
"Fixing" this will require adjusting the background scrolling algorithm.

Reply:
Huh, I never run on actual 1920x1080 so I never noticed the black bar. I'd agree that the bitmap size should be based on the resolution and the scrolling and calculated automatically or something, but that's probably easier said than done. Would you mind chucking in an issue (or, if you wanna fix this sooner rather than later, don't bother) to fix this little bug?

Reply:
Sure, #199.